website/docs/components/page-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,237 @@
+## Usage
+
+### When to use
+
+- To display the title or topmost heading within a page.
+- To communicate metadata pertaining to the page; e.g., status, description, subtitle, etc.
+- To highlight page-level actions and functions.
+
+### When not to use
+
+- To communicate page-level information anywhere other than the top of the page.
+
+## Title
+
+Displays the title or topmost heading of the page. This should be limited to no more than four words and not exceed 50 characters.
+
+### Multiple `<h1>` elements
+
+Since the Page Header renders the title in an `<h1>` element, ensure that this is the only `<h1>` on the page. More than one `<h1>` can confuse screen readers and impact the semantic hierarchy of the page.
+
+## Size
+
+Size only pertains to the Figma component and accounts for smaller viewports by stacking elements. The Ember component supports a basic level of fluidity and responsiveness out of the box.
+
+Refer to [responsiveness](#responsiveness) for more information on how the component adapts to different viewport and container sizes.
+
+### Large
+
+![Large Page Header variant](/assets/components/page-header/page-header-size-large.png =945x*)
+
+### Small
+
+![Small Page Header variant](/assets/components/page-header/page-header-size-small.png =300x*)
```

Comment:
I've nested the section on responsiveness under the size since they are related and moved it to the end since it's less relevant than the other topics.